gdp/small_batch/gdp_small_batch.py

```diff
@@ -1,112 +1,300 @@
+    )
+
+    s_init = {
+        ('a', 'mixer'): 2,
+        ('a', 'reactor'): 3,
+        ('a', 'centrifuge'): 4,
+        ('b', 'mixer'): 4,
+        ('b', 'reactor'): 6,
+        ('b', 'centrifuge'): 3,
+    }
+
+    # Size factor for product i in stage j [kg/L]
+    m.s = pe.Param(
+        m.i, m.j, initialize=s_init, doc='Size factor for product i in stage j [kg/L]'
+    )
+
+    t_init = {
+        ('a', 'mixer'): 8,
+        ('a', 'reactor'): 20,
+        ('a', 'centrifuge'): 4,
+        ('b', 'mixer'): 10,
+        ('b', 'reactor'): 12,
+        ('b', 'centrifuge'): 3,
+    }
+
+    # Processing time of product i in batch j [hr]
+    m.t = pe.Param(
+        m.i, m.j, initialize=t_init, doc='Processing time of product i in batch j [hr]'
+    )
 
     # Variables
-    m.Y = pe.BooleanVar(m.k, m.j)    # Stage existence
-    m.coeffval = pe.Var(m.k, m.j,  within=pe.NonNegativeReals,
-                        bounds=(0, pe.log(NK)))  # Activation of coeff
-    m.v = pe.Var(m.j, within=pe.NonNegativeReals, bounds=(
-        pe.log(m.vlow), pe.log(m.vupp)))  # Volume of stage j
-    m.b = pe.Var(m.i, within=pe.NonNegativeReals)  # Batch size of product i
-    m.tl = pe.Var(m.i, within=pe.NonNegativeReals)  # Cycle time of product i
+    m.Y = pe.BooleanVar(m.k, m.j, doc='Stage existence')  # Stage existence
+    m.coeffval = pe.Var(
+        m.k,
+        m.j,
+        within=pe.NonNegativeReals,
+        bounds=(0, pe.log(NK)),
+        doc='Activation of Coefficient',
+    )  # Activation of coeff
+    m.v = pe.Var(
+        m.j,
+        within=pe.NonNegativeReals,
+        bounds=(pe.log(m.vlow), pe.log(m.vupp)),
+        doc='Colume of stage j [L]',
+    )  # Volume of stage j [L]
+    m.b = pe.Var(
+        m.i, within=pe.NonNegativeReals, doc='Batch size of product i [L]'
+    )  # Batch size of product i [L]
+    m.tl = pe.Var(
+        m.i, within=pe.NonNegativeReals, doc='Cycle time of product i [hr]'
+    )  # Cycle time of product i [hr]
     # Number of units in parallel stage j
-    m.n = pe.Var(m.j, within=pe.NonNegativeReals)
+    m.n = pe.Var(
+        m.j, within=pe.NonNegativeReals, doc='Number of units in parallel stage j'
+    )
 
     # Constraints
 
     # Volume requirement in stage j
     @m.Constraint(m.i, m.j)
     def vol(m, i, j):
+        """
+        Volume Requirement for Stage j.
+        Equation:
+            m.v[j] >= log(m.s[i,j]) + m.b[i]
+
+        Args:
+            m (pyomo.ConcreteModel): small batch GDP model
+            i (str): product
+            j (str): stage
+
+        Returns:
+            Algebraic Constraint
+        """
         return m.v[j] >= pe.log(m.s[i, j]) + m.b[i]
 
     # Cycle time for each product i
     @m.Constraint(m.i, m.j)
     def cycle(m, i, j):
+        """
+        Cycle time for each product i.
+        Equation:
+            m.tl[i] >= log(m.t[i,j])
+
+        Args:
+            m (pyomo.ConcreteModel): small batch GDP model
+            i (str): product
```

Comment:
These are not just string but within the sets. Specify in the description (`'a' or 'b'`). Siomilar with `j`